diff --git a/include/configs/zynq-common.h b/include/configs/zynq-common.h
index 6c37909..f1ffec8 100644
--- a/include/configs/zynq-common.h
+++ b/include/configs/zynq-common.h
@@ -240,7 +240,6 @@
 
 /* Default environment */
 #define CONFIG_EXTRA_ENV_SETTINGS	\
-	"ethaddr=00:0a:35:00:01:22\0"	\
 	"kernel_image=uImage\0"	\
 	"kernel_load_address=0x2080000\0" \
 	"ramdisk_image=uramdisk.image.gz\0"	\
@@ -338,8 +337,6 @@
 		DFU_ALT_INFO
 
 /* Default environment */
-#define CONFIG_IPADDR	10.10.70.102
-#define CONFIG_SERVERIP	10.10.70.101
 
 /* default boot is according to the bootmode switch settings */
 #if defined(CONFIG_CMD_ZYNQ_RSA)
